README.md

```diff
@@ -115,7 +116,16 @@ You must specify the `CLUSTER_NAME` environment variable. `NAMESPACE` and `DEFAU
     ```
 
 ## Usage
-Add the `eip.aws.materialize.com/manage=true` label to any pods that you want this to manage.
+Instantiate an Eip Kubernetes object, specifying the `podName` in the spec.
+```yaml
+apiVersion: "cloud.materialize.com/v1"
+kind: Eip
+metadata:
+  name: my-new-eip
+spec:
+  podName: my-pod
```

Comment:
My first reaction here was that this maybe should be a `podSelector`, but then you can only ever assign an EIP to a single pod (never several), so that seems reasonable.

Might be worth documenting how the association from the concrete EIP to this custom resource happens, though: AIUI, it's done via the resource name? (Also, probs worth mentioning that the association from this object's name to the AWS EIP object happens via tags on the EIP itself)

Reply:
The K8S Eip resource and the AWS EIP are associated via tags on the AWS EIP object (specifically using the K8S Eip object's uid) and via status fields on the K8S Eip object for the allocation ID and public IP.

I can add this to the docs.

The name of the K8S Eip resource is not used by the eip-operator.